src/wp-admin/options-media.php

```diff
@@ -44,6 +44,17 @@
 
 require_once ABSPATH . 'wp-admin/admin-header.php';
 
+/**
+ * Get upload organization preference.
+ *
+ * @since CP-2.1.0
+ *
+ * New option based on year only.
+ */
+$storefolders = '0';
```

Comment:
The option is not showing up. Maybe change this code to:
`$storefolders = get_option( 'uploads_use_yearmonth_folders', '1' );`

Reply:
Then the following `if` statement is no longer required.

Reply:
No, that would make it always use year-and-month folders. The solution is to change the code in the line you've commented on below.

Reply:
Ok, now understood!